src/m365/spfx/commands/spfx-doctor.ts

```diff
@@ -793,6 +687,14 @@ class SpfxDoctorCommand extends AnonymousCommand {
         fixes.push(versionCheck.fix);
       }
     }
+    // else {
+    //   TODO: we might need this code in the future if SPFx introduces required
+    //   prerequisites with a specific version
+    //   if (optionalOrRequired === OptionalOrRequired.Required) {
+    //     logger.log(this.getStatus(CheckStatus.Failure, `${what} not found, v${versionCheck.range} required`));
+    //     fixes.push(versionCheck.fix);
+    //   }
+    // }
   }
```

Comment:
I chose to invert this `if` statement because after deleting the react tests, I noticed that the commented out code in the `if` branch had no code coverage anymore. I found it a bit weird to write a test just to cover commented out code. That's why I chose to invert the `if` and comment out the `else` branch.